scripts/build.sh

```
@@ -22,13 +24,16 @@ cp target/release/zrc "$ZIRCON_BIN_DIR/"
 cp target/release/zircop "$ZIRCON_BIN_DIR/"
 cp target/release/zrx "$ZIRCON_BIN_DIR/"
 cp target/release/zrepl "$ZIRCON_BIN_DIR/"
+cp target/release/libzrc.a "$ZIRCON_LIB_DIR/"
+find target/release -maxdepth 1 -type f \( -name "libzrc.so" -o -name "libzrc.dylib" \) -exec cp {} "$ZIRCON_LIB_DIR/" \;
 cp -r include/* "$ZIRCON_INCLUDE_DIR/"
+cp compiler/libzrc/zrc.h "$ZIRCON_INCLUDE_DIR/"
 
 # build std using the fresh compiler
 make -C libzr all-opt ZRC=$ZIRCON_BIN_DIR/zrc
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)